service/docs/source/overview.rst

```diff
@@ -803,11 +803,13 @@ GEOPM Runtime features.  The first is creating a CSV trace file using
 the ``GEOPM_TRACE`` environment variable.  Additionally, by using the
 ``GEOPM_PERIOD`` environment variable we increase the sampling period
 of the controller to 200 milliseconds (default value is 5
-milliseconds).  These two options together will create a CSV trace
-file with approximately 50 rows of samples (five per-second for ten
-seconds).  The ``awk`` command in the example selects the columns
-measuring time since application start from column 1, CPU energy from
-column 6, and CPU power from column 8.
+milliseconds). By using the optional ``GEOPM_PROGRAM_FILTER`` variable
+we can explicitly list the name of the non-MPI program invovation name
+of the non-MPI process to be profiled. These two options together will 
```

Comment:
You might need to rearrange this a bit to avoid confusion. The new sentence that starts in line 808 says `These two options...` in reference to `GEOPM_TRACE` and `GEOPM_PERIOD`, so mentioning `GEOPM_PROGRAM_FILTER` right before this could generate confusion. I'm thinking you could simply move the `GEOPM_PROGRAM_FILTER` sentence after that one.